client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-type/style.scss

```diff
@@ -0,0 +1,24 @@
+@import "../style";
+
+.newsletter-type {
```

Comment:
Most of this CSS is duplicate from setup step, which tells me that we should probably componentise it and stop copying across steps.